website/blog/2018-07-14-whats-the-deal-with-the-new-pipeline-plugin-configuration.md

```diff
@@ -0,0 +1,153 @@
+
+## Where Babel Comes In
+
+Once all three proposals were fleshed out, we realized discussion and debate were unlikely to resolve the inherent tension between them. We decided the best way to make a decision would be with the feedback of developers, using the proposals in real code. Given babel's role in the community, we approached them about including all three of the above proposals into the pipeline operator plugin.
+
+Because these proposals parse slightly differently, support needs to be added to `@babel/parser` (formerly `babylon`), and the parser needs to be configured based on which proposal is its current parse target. The pipeline operator plugin itself thus needs the `"proposal"` option, both to configure babylon as well as its own transformation.
+
+We were working under a compressed timeline for this, as we needed to introduce any breaking changes to babel, `@babel/parser`, as well as the pipeline proposal plugin before babel@7 left beta. We also wanted the plugin to be able to eventually default to whichever of the proposals get accepted, so the configuration option becomes obsolete. Given these two constraints, we opted to introduce this new configuration option and make it required, forcing users to decide which proposal they want to use in their project.
+
+Because the pipeline proposal is currently stage-1, both the `stage-1` & `stage-0` plugins include the pipeline operator. An option was added to both of those plugins, `"pipelineProposal"`, to forward the value to the pipeline plugin. This option was also required, and thus breaking, so it caused some frustration for some users. The stage presets have been deprecated and [are being removed](https://github.com/babel/babel/pull/8293), because when those presets are used, consumers often don't know what plugins they're including, so changes like this are surprising and frustrating.
+
+Once a particular proposal gets accepted as the canonical behavior of the pipeline operator, we'll deprecate the `"proposal"` option and make the default whichever gets accepted, and the rejected proposals will be removed in the following major version.
+
+## Get Involved
+
+If you're interested in getting involved in the pipeline proposal, all the conversations are public and can be found in the [pipeline operator repository](https://github.com/tc39/proposal-pipeline-operator/). You can also check out the [presentation from the last TC39 meeting](https://docs.google.com/presentation/d/1eFFRK1wLIazIuK0F6fY974OIDvvWXS890XAMB59PUBA/edit#slide=id.p). Lastly, you can find [James DiGioia](https://twitter.com/jamesdigioia), [J. S. Choi](https://twitter.com/__jschoi), or [Daniel Ehrenberg](https://twitter.com/littledan) on Twitter.
+
+But most importantly, once the work is complete, try out the pipeline operator in your own projects! We need feedback and real code in order for this to be useful, so we'd love to hear from you.
```

Comment:
We can mention here we'll be updating the REPL so that you can test each proposal there as options, and then you can experiment online and share/provide feedback. Also can just say you can ping the `@babeljs` twitter maybe as well.